apps/ima/behaviours/continue-report.js

```diff
@@ -64,12 +80,22 @@ module.exports = superclass => class extends superclass {
   }
 
   saveValues(req, res, next) {
-    super.saveValues(req, res, err => {
+    super.saveValues(req, res, async err => {
       if (err) {
         next(err);
       }
       req.sessionModel.set('redirect-to-summary', false);
       if (req.body['save-and-exit']) {
+        const host = req.get('host');
+        const userEmail = req.form.values['user-email'] || req.sessionModel.get('user-email');
+        const token = tokenGenerator.save(req, userEmail);
+        try {
+          await notifyClient.sendEmail(templateId, userEmail, {
+            personalisation: getPersonalisation(host, token, req, req)
+          });
+        } catch (e) {
+          return next(e);
+        }
```

Comment:
This is really picky but you've used `err` above and `e` here to represent the error. Either is fine of course, but I'd ideally stick with the same through a method or func.

Reply:
Using `err` instead of `e` throws the following linting error: `'err' is already declared in the upper scope on line...` so I have left this unchanged

Reply:
Cool, no problem then